common/cards/effects/berry-bush.ts

```diff
@@ -0,0 +1,114 @@
+import EffectCard from '..//base/effect-card'
+import {isTargetingPos} from '../../utils/attacks'
+import {GameModel} from '../../models/game-model'
+import {discardCard} from '../../utils/movement'
+import {CardPosModel, getBasicCardPos} from '../../models/card-pos-model'
+import {TurnActions} from '../../types/game-state'
+
+class BerryBushEffectCard extends EffectCard {
+	constructor() {
+		super({
+			id: 'berry_bush',
+			numericId: 173,
```

Comment:
Numeric id needs updating
```suggestion
			numericId: 176,
```